components/cluster/pacemaker/Makefile

```diff
@@ -43,16 +42,14 @@ CONFIGURE_SCRIPT= $(@D)/configure
 CFLAGS += -D__EXTENSIONS__ -D_POSIX_PTHREAD_SEMANTICS -lssp_nonshared -lssp -lnsl
 
 # Force use of gnutls-3 pkgconfig during 2.x->3.x transition
-GNUTLS_PKG_CONFIG_PATH_32 = /usr/lib/pkgconfig/gnutls-3
 GNUTLS_PKG_CONFIG_PATH_64 = /usr/lib/$(MACH64)/pkgconfig/gnutls-3
 PKG_CONFIG_PATH = $(GNUTLS_PKG_CONFIG_PATH_$(BITS)):$(PKG_CONFIG_PATH.$(BITS))
 
-LD_OPTIONS.32 += -L/usr/lib/gnutls-3 -R/usr/lib/gnutls-3
 LD_OPTIONS.64 += -L/usr/lib/$(MACH64)/gnutls-3 -R/usr/lib/$(MACH64)/gnutls-3
 LD_OPTIONS += $(LD_OPTIONS.$(BITS))
 
+CONFIGURE_OPTIONS.64 += --libexecdir=$(CONFIGURE_PREFIX)/lib/$(MACH64)
```

Comment:
What happens if we set it to $(CONFIGURE_PREFIX)/lib ? What files does it affect?

Reply:
I guess this would be unnecessary (untested!):
file usr/lib/$(MACH64)/pacemaker/attrd             path=usr/lib/pacemaker/attrd
file usr/lib/$(MACH64)/pacemaker/cib               path=usr/lib/pacemaker/cib
file usr/lib/$(MACH64)/pacemaker/cibmon            path=usr/lib/pacemaker/cibmon
file usr/lib/$(MACH64)/pacemaker/crmd              path=usr/lib/pacemaker/crmd
file usr/lib/$(MACH64)/pacemaker/lrmd              path=usr/lib/pacemaker/lrmd
file usr/lib/$(MACH64)/pacemaker/lrmd_test         path=usr/lib/pacemaker/lrmd_test
file usr/lib/$(MACH64)/pacemaker/pengine           path=usr/lib/pacemaker/pengine
file usr/lib/$(MACH64)/pacemaker/stonith-test      path=usr/lib/pacemaker/stonith-test
file usr/lib/$(MACH64)/pacemaker/stonithd          path=usr/lib/pacemaker/stonithd
file usr/lib/$(MACH64)/pacemaker/lrmd_internal_ctl path=usr/lib/pacemaker/lrmd_internal_ctl